README.md

```diff
@@ -1 +1,122 @@
-# School_library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>School_library</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👤 Author](#author)
+- - [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 school_library <a name="about-project"></a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>RUBY</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Module and Class**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/tajulafreen/School_library.git
+  cd School_library
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤 Author <a name="author"></a>
+
+👤 **Yonas Henok**
+
+- GitHub: [@YonasHenok](https://github.com/tajulafreen)
+- Twitter: [@YonasHenok3](https://www.twitter.com/tajulafreen)
+- LinkedIn: [Yonas Henok](https://www.linkedin.com/in/tajul-afreen/)
```

Comment:
- [ ] [OPTIONAL] Kindly update the name in the "Authors" section to your proper name as it might cause confusion and make your repo look unprofessional. 👶